org.lflang/src/org/lflang/generator/TargetTypes.java

```diff
@@ -68,6 +69,12 @@ default String getTargetParamRef(ParameterReference expr, InferredType typeOrNul
         return escapeIdentifier(expr.getParameter().getName());
     }
 
+    /** Translate the braced list expression into target language syntax. */
+    default String getTargetBracedListExpr(BracedListExpression expr, InferredType typeOrNull) {
```

Comment:
I wonder if the `TargetTypes` class is the right place for this, but I am Ok with keeping it here for now. I think will need something like a `ExpressionGenerator` sooner or later (this is where the expression visitor will be very useful).

Reply:
Yes I think you're right that this class is getting too complex, we probably can split it later using an expression visitor